AlphaWallet/Transfer/Controllers/TransactionConfigurator.swift

```diff
@@ -128,7 +128,8 @@ class TransactionConfigurator {
                                 // Guard against really high prices
                                 seal.fulfill(GasPriceConfiguration.maxPrice)
                             } else {
-                                seal.fulfill(gasPrice)
+                                //Add an extra gwei because the estimate is sometimes too low
+                                seal.fulfill(gasPrice + BigInt(EthereumUnit.gwei.rawValue))
```

Comment:
Good to make a constant in `GasPriceConfiguration.oneGwei = EthereumUnit.gwei.rawValue`

Also need to check that it isn't bigger than the max after + 1

Reply:
Hmm? I thought that this is implicit given that gwei is like eth. You would assume one by default no?

Reply:
Took a bit more effort to parse and verify is correct because I read `.rawValue` as `1_000_000_000` rather than 1 gwei.